packages/vite/src/node/cli.ts

```diff
@@ -96,24 +96,26 @@ cli
 
       const info = server.config.logger.info
 
+      const { version } = require('vite/package.json')
+
+      // @ts-ignore
+      const viteStartTime = global.__vite_start_time ?? false
+      const startupDurationString = viteStartTime
+        ? colors.dim(
+            `(ready in ${colors.white(
+              colors.bold(Math.ceil(performance.now() - viteStartTime))
+            )} ms)`
```

Comment:
Since it's already dimming, maybe we could omit the parentheses here to make it more concise?

Reply:
Thanks! Fine by me, seems like a good improvement. 👍